### PR TITLE
Add description of env variables and endpoints

### PR DESCRIPTION
**Pull request by vermakhushboo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,13 +122,74 @@ curl -H "authorization: Bearer executor-secret-key" -H "Content-Type: applicatio
...
+| `destination` | `string` | Destination folder to store runtime files into |  |
+| `variables` | `json` | Environment variables passed into runtime |  |
+| `runtimeEntrypoint` | `string` | Commands to run when creating a container. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `command` | `string` | Commands to run after container is created. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `timeout` | `string` | Commands execution time in seconds |  |
+| `cpus` | `integer` | Container CPU |  |
+| `memory` | `integer` | Container RAM memory |  |
+| `version` | `array` | Runtime Open Runtime version |  |
+
+### /v1/runtimes/{runtimeId}/executions
```
Comment (christyjacob4) [inline README.md:159]: Is there a reason why we have only documented 3 out of the 7 endpoints?

Comment (vermakhushboo) [reply README.md:164]: Those endpoints don't have any params. Updated the params column with X for those.

File: README.md [hunk 2]
```diff
@@ -122,13 +122,79 @@ curl -H "authorization: Bearer executor-secret-key" -H "Content-Type: applicatio
...
+| GET |`/v1/runtimes/{runtimeId}/logs`| Get live stream of logs of a runtime | [JSON](#v1runtimesruntimeidlogs) |
+| POST |`/v1/runtimes`| Create a new runtime server | [JSON](#v1runtimes) |
+| GET |`/v1/runtimes`| List currently active runtimes | X |
+| GET |`/v1/runtimes/{runtimeId}`| Get a runtime by its ID | [JSON](#v1runtimesruntimeid) |
+| DELETE |`/v1/runtimes/{runtimeId}`| Delete a runtime | [JSON](#v1runtimesruntimeid) |
+| POST |`/v1/runtimes/{runtimeId}/executions`| Create an execution | [JSON](#v1runtimesruntimeidexecutions) |
+| GET |`/v1/health`| Get health status of host machine and runtimes | X |
+
+### /v1/runtimes/{runtimeId}/logs
+| Param | Type | Description | Required |
```
Comment (Meldiron) [inline README.md:138]: We could extend table by 1 more column, `default`. If required=No, then showing default is valuable

File: README.md [hunk 3]
```diff
@@ -122,13 +122,79 @@ curl -H "authorization: Bearer executor-secret-key" -H "Content-Type: applicatio
...
+| `entrypoint` | `string` | Entrypoint of the code file |  |
+| `source` | `string` | Path to source files |  |
+| `destination` | `string` | Destination folder to store runtime files into |  |
+| `variables` | `json` | Environment variables passed into runtime |  |
+| `runtimeEntrypoint` | `string` | Commands to run when creating a container. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `command` | `string` | Commands to run after container is created. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `timeout` | `string` | Commands execution time in seconds |  |
+| `cpus` | `integer` | Container CPU |  |
+| `memory` | `integer` | Container RAM memory |  |
+| `version` | `array` | Runtime Open Runtime version |  |
```
Comment (Meldiron) [inline README.md:157]: Not array, string. Also lets show possible values.

File: README.md [hunk 4]
```diff
@@ -122,13 +122,79 @@ curl -H "authorization: Bearer executor-secret-key" -H "Content-Type: applicatio
...
+| `image` | `string` | Base image name of the runtime | ✅ |
+| `entrypoint` | `string` | Entrypoint of the code file |  |
+| `source` | `string` | Path to source files |  |
+| `destination` | `string` | Destination folder to store runtime files into |  |
+| `variables` | `json` | Environment variables passed into runtime |  |
+| `runtimeEntrypoint` | `string` | Commands to run when creating a container. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `command` | `string` | Commands to run after container is created. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `timeout` | `string` | Commands execution time in seconds |  |
+| `cpus` | `integer` | Container CPU |  |
+| `memory` | `integer` | Container RAM memory |  |
```
Comment (Meldiron) [inline README.md:156]: Lets mention unit

File: README.md [hunk 5]
```diff
@@ -122,13 +122,79 @@ curl -H "authorization: Bearer executor-secret-key" -H "Content-Type: applicatio
...
+| `runtimeId` | `string` | Runtime unique ID | ✅ |
+| `image` | `string` | Base image name of the runtime | ✅ |
+| `entrypoint` | `string` | Entrypoint of the code file |  |
+| `source` | `string` | Path to source files |  |
+| `destination` | `string` | Destination folder to store runtime files into |  |
+| `variables` | `json` | Environment variables passed into runtime |  |
+| `runtimeEntrypoint` | `string` | Commands to run when creating a container. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `command` | `string` | Commands to run after container is created. Maximum of 100 commands are allowed, each 1024 characters long. |  |
+| `timeout` | `string` | Commands execution time in seconds |  |
+| `cpus` | `integer` | Container CPU |  |
```
Comment (Meldiron) [inline README.md:155]: Lets have sample value

File: README.md (hunk 4 above)
Comment (Meldiron) [inline README.md:156]: Lets have better descriptions here, like `Maximum CPU cores runtime can utilize`

File: README.md [hunk 6]
```diff
@@ -122,13 +122,79 @@ curl -H "authorization: Bearer executor-secret-key" -H "Content-Type: applicatio
...
+|-------|------|-------------|----------|
+| `runtimeId` | `string` | Runtime unique ID | ✅ |
+
+### /v1/runtimes/{runtimeId}/executions
+| Param | Type | Description | Required |
+|-------|------|-------------|----------|
+| `runtimeId` | `string` | The runtimeID to execute | ✅ |
+| `body` | `string` | Data to be forwarded to the function, this is user specified. |  |
+| `path` | `string` | Path from which execution comes |  |
+| `method` | `array` | Path from which execution comes |  |
```
Comment (Meldiron) [inline README.md:170]: String I think

**Review by Meldiron — APPROVED** [review]

**Pull request merged**